ansible/templates/minigraph_dpg.j2

```diff
@@ -32,9 +32,9 @@
           <Name>V6HostIP</Name>
           <AttachTo>eth0</AttachTo>
           <a:Prefix xmlns:b="Microsoft.Search.Autopilot.Evolution">
-            <b:IPPrefix>{{ ansible_hostv6 }}/64</b:IPPrefix>
+            <b:IPPrefix>{{ ansible_hostv6 if ansible_hostv6 is defined else 'FC00:2::32' }}/64</b:IPPrefix>
           </a:Prefix>
-          <a:PrefixStr>{{ ansible_hostv6 }}/64</a:PrefixStr>
+          <a:PrefixStr>{{ ansible_hostv6 if ansible_hostv6 is defined else 'FC00:2::32' }}/64</a:PrefixStr>
```

Comment:
Do we need to be aware that if ansible_hostv6 is not defined, then tacacs v6 test will fail?

Reply:
@SuvarnaMeenakshi , can you answer this question? If yes, then how do we know whether we should run tacacs v6 test case or not?

Reply:
Currently, we check if ptf has IPv6 address assigned or not. If ptf does not have IPv6 address assigned, then tacacs ipv6 test case will be skipped.
If ptf has IPv6 assigned and minigraph does not have IPv6, tacacs ipv6 will be executed, and it will fail.
Should we have additional check to see if both ptf and DUT has IPv6 configured, only then execute tacacs ipv6 test?

Reply:
Sorry for my 2 cents, I suppose we need to have backward compatibility, run deploy-mg with/without `ansible_hostv6` definition

Reply:
@NazarTkachuk ,  I am trying understand what else changes we need so that we are not breaking the tacacs v6 test.

Reply:
@SuvarnaMeenakshi , in this case, the dut will have ipv6 address. How do you check?

Reply:
@lguohan, that is right, we will have IPv6 address in this case, can think of 2 options: 1.	Expect that if PTF docker has IPv6 address, then the DUT has IPv6 address reachable from ptf docker – If that is the case, then current implementation is good.
2.	Specifically check for the default IPv6 address given to DUT here, if it is the default address, then skip IPv6 tacacs test. This does not seem clean as it is specific check for address: 'FC00:2::32'.

Reply:
can we change the default v6 address on DUT a IPv6 address that is reachable to the ptf docker?

Reply:
@lguohan We could do that for VS testbed, if ptf-docker IP address is taken from a specific lab network, then we cannot generate a default reachable IPv6 address for DUT.